addons/stock/views/stock_picking_views.xml

```diff
@@ -324,7 +324,7 @@
                                         <field name="product_qty" invisible="1" readonly="1"/>
                                         <field name="forecast_expected_date" invisible="1"/>
                                         <field name="forecast_availability" string="Reserved" attrs="{'invisible': ['|', ('parent.picking_type_code', '!=', 'outgoing'), ('parent.state','=', 'done')]}" widget="forecast_widget"/>
-                                        <field name="quantity_done" string="Done" attrs="{'readonly': [('is_quantity_done_editable', '=', False)]}"/>
+                                        <field name="quantity_done" string="Done" attrs="{'readonly': [('product_id', '=', False)]}"/>
```

Comment:
Also need to remove the `is_quantity_done_editable` field in the form since we don't need it anymore.

Reply:
I would keep it in case of custo